doc/source/advanced.md

```diff
@@ -75,6 +75,70 @@ and **google cloud's ingress controller**.
 This should provision a certificate, and keep renewing it whenever it gets close
 to expiry!
 
+
+## Using PostgreSQL as database for you Hub.
+
+Four types of settings are allowed for storing hub state, as described in
+the key ``hub.db.type``:
+
+- ``sqlite-pvc``
+- ``sqlite-memory``
+- ``mysql``
+- ``postgres``
+
+Using PostgreSql or MySQL provides the main advantage to allow seemless update
+of your hub without discontinuity of service (logged in users can always
+continue to work on running server, even with SQLite, but the difference between
+SQLite and MySQL/PostgreSQL are for new user, in order to allow them to log in
+even when a new hub is starting).
+
+Please note running PostgreSQL on Kubernetes is hard. You can find an lot of
+detailed articles on the web about this suject, such as: 
+[PostgreSQL on Kubernetes the Right Way](https://medium.com/kokster/postgresql-on-kubernetes-the-right-way-part-one-d174ee8a56e3)
+
+It is advised to use the Database as a Service solution that your Cloud
+provider may provide you.
+
+You have the option to run PostgreSQL for the Hub using Helm 'stable/postgresql'
```

Comment:
Can we instead provide content like:

> You can run postgresql either via your cloud provider's hosted offering (linkes to Amazon RDS, Google Cloud SQL, Azure's Database), or by setting it up yourself manually. There's experimental method of running it directly on kubernetes (link to helm chart), but we do not recommend it.

I'd prefer we have that rather than provide direct instructions on using the helm chart here.